content/docs/others/streamlining-server-deployment-with-github-actions.md

```diff
@@ -0,0 +1,867 @@
+---
+title: "Streamlining Server Deployment with Github Actions"
+description: ""
+lead: "Author/s: Gab Barbudo"
+date: 2023-09-13T13:43:10+08:00
+draft: false
+menu: 
+  docs:
+    parent: "others"
+weight: 999
+toc: true
+---
+## Introduction
+
+### 1. Problem
+
+Deploying code to development servers can be a time-consuming and error-prone process when done manually. As of this writing, we have a total of 11 development servers, each requiring the following steps for deployment:
+
+- Make changes to the codebase
+- Push the changes to GitHub
+- SSH into each individual dev server
+- Pull the latest changes from GitHub
+- Depending on the nature of the changes, execute `bench migrate`, `bench build` or `bench import-initial-customization`
```

Comment:
How about `bench restart`?

Reply:
`bench restart` is not available in development mode and is also not needed since `bench` automatically restarts or refreshes when there are changes in python files